core/platform/image/gae_image_services_test.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for methods in the gae_image_services."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+
+from constants import constants
+from core.platform.image import gae_image_services
+from core.tests import test_utils
+import feconf
+import python_utils
+
+
+class GaeImageServicesUnitTests(test_utils.GenericTestBase):
+    """Tests for gae_image_services."""
+
+    TEST_IMAGE_WIDTH = 3000
+    TEST_IMAGE_HEIGHT = 2092
+
+    def setUp(self):
+        super(GaeImageServicesUnitTests, self).setUp()
+        with python_utils.open_file(
+            os.path.join(feconf.TESTS_DATA_DIR, 'dummyLargeImage.jpg'),
+            mode='rb', encoding=None) as f:
+            self.raw_image = f.read()
+
+    def test_get_image_dimensions(self):
```

Comment:
Here and below: please follow the guidelines in point 2 [here](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests) for writing good test names.

Reply:
Done